### ISSUE TITLE
Idea-- Hot Take

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#4107]
name is TBD, but for now, i'll just call it a HotTake.

a HotTake is a group of heterogenous entities:
- it includes all the work you've done across different platforms (github, notion, gdocs, etc)
- it can also include any tasks created or updates to tasks
- it includes audio, video, or text, which can be used as a way to update the team, ask for help, bring up a tension
- it includes links to other hot takes by you or by others (e.g. oppose another hot take, build on it, or otherwise reference it)

Hot Takes are the primary entity in a stand up meeting. 
- Each team member can create one or more for a meeting
- The rest of the team can view the hot takes. 
- Popular ones get recirculated outside of their immediate team.
- Managers can create hot takes based off of their team's hot takes, plucking out individual pieces. However, the link to the original cannot be broken.

Hot Takes have a medium shelf life
- They require more effort than a message in slack
- They require much less effort than a decision or policy
- Big picture, I don't want low friction items (e.g. slack communications) to live in Parabol. 
- Once in parabol, you're guaranteed a certain level of fidelity. Higher fidelity items (policies, decisions, etc.) can be built on these lower fidelity items

How it's implemented:
- When you log into Parabol, you land on a timeline
- You see things from across your organization. 80% your team, 10% your manager's team, 10% random across the organization
- As you upvote & interact with items, their score increases & they are seen by others
- when you create one (which is probably the CTA of the timeline) you can build off of others
- big picture, the goal is to remove the work of the manager. managers see things flow through & can vote on them & they good ones make it up to their managers

**Comment by jordanh** [comment]
Now this is a Hot Take™

**Comment by ackernaut** [comment]
I dig it and can’t wait to further explore this form of mid-weight, rich, async sharing

**Comment by iamjoshfrank** [comment]
I really like the ingenuity in this Hot Take.

There are several components of this Hot Take idea that overlap with what I've been noodling!
- non-text-based media
- integrated tasks from multiple sources
- "build your standup" from existing + new parts
- how other team members interact (or don't) with your Standup (or its component parts)
- the possibility of interaction / "social sharing" mechanics

Where I'm spending solo cycles:
- getting more concrete about the problems, tensions, and jobs we're hiring Standups to resolve
- connecting some of the nascent ideas to these ^^

The only "goal" I see concretely spelled out in this Hot Take is:

> big picture, the goal is to remove the work of the manager. managers see things flow through & can vote on them & they good ones make it up to their managers.

I'd like context, goals, and before-and-after state to come into more focus.

@mattkrick - a few questions:
1. how would you describe the setting for this idea (the setting / scenario you imagine users being in)?
2. what motivated you to compose this idea?
3. what outcomes do you think this idea is trying to derive?

**Comment by iamjoshfrank** [comment]
A few other thoughts:

1. I think we have to consider heavily the new value being created, the tension being resolved, or the conflict being mitigated for the Team Contributor*. These TCs are the ones that would be expected to create the greatest number of Hot Takes. What's in it for them?
2. IMHO, it's really easy to think through the lens of the Manager when considering Standups. Let's hear some more about "the work" that we're eliminating / removing for the manager:

> remove the work of the manager

3. Where do the Accountants / Billing / Spending folks fit in? How do we describe the value to them (is there any)?

---

_*I know it might be semantics, but the term "Individual Contributors" always gets under my skin. So I'm opting for "Team Contributors" instead._

**Comment by mattkrick** [comment]
> how would you describe the setting for this idea (the setting / scenario you imagine users being in)?

unspecified. we need to avoid coming off like range.co or a slackbot. nagging users with "what are you working on?" feels wrong. the billion dollar question is how do we encourage users to generate content without their managers telling them to.

> what motivated you to compose this idea?

B2B lags behind C2C. tiktok conquered the C2C so we don't really need to reinvent the wheel here, we just need to see what works in the consumer world & bring it to the business world while putting some fallbacks in place for the luddites (e.g. allow textual updates instead of video)
 
> what outcomes do you think this idea is trying to drive?

daily engagement, a deliverable that can live outside of parabol. if parabol can give managers a 2 minute highlight reel that they can give to their boss to make them look good, we win.

> I think we have to consider heavily the new value being created, the tension being resolved, or the conflict being mitigated for the Team Contributor*. These TCs are the ones that would be expected to create the greatest number of Hot Takes. What's in it for them?

Community. In a remote-first world, people want to be seen, be recognized for their good work. we need to leverage that desire for attention and affirmation. posting the content _must_ be its own reward. fucked up as it is, social media nurtures a dependence for a single dose of attention to keep folks coming back. we have a leg up over the consumer market because when you create an instagram, you have 0 followers & you follow 0 people. Here, you follow & are followed by your entire org. we must leverage that instant network.

> IMHO, it's really easy to think through the lens of the Manager when considering Standups. Let's hear some more about "the work" that we're eliminating / removing for the manager:

most managers suck at their job. they followed the peter principle & have imposter syndrome. we can leverage that by making them look good by giving them a slick deliverable. if they look good, we get paid.

> Where do the Accountants / Billing / Spending folks fit in? How do we describe the value to them (is there any)?

target market is product/project teams within tech. these are the early adopters. we should be laser focused on our target market, get them stoked, and only after we've crushed it should we expand to other demographics. we'll know we're there when orgs that use us start experimenting with us in those HR & finance teams.

**Comment by iamjoshfrank** [comment]
> Where do the Accountants / Billing / Spending folks fit in? How do we describe the value to them (is there any)?

>> target market is product/project teams within tech. these are the early adopters. we should be laser focused on our target market, get them stoked, and only after we've crushed it should we expand to other demographics. we'll know we're there when orgs that use us start experimenting with us in those HR & finance teams.

Quick clarification: totally agree re: target market. This question came more from the, "for the people actually approving/buying the software, (how) does the Hot Take demonstrate purchasable value to them?"

From our persona research discussion on Tuesday, we know that the folks with purchasing power don't really give two shits about tools for agile. So, I'm pushing to inspect this idea from the standpoint of, "how does this motivate a buyer to approve / make the purchase?"

Now, if there's radical bottom-up adoption of a viral feature / product, then the purchase approval might just be a no-brainer.

---

I think what I'm trying to tease out here is about the value streams, about "jobs to be done" (without necessarily being married to JTBD as a framework, cuz I'm not 😉 ).

I'll have some more to share about that maybe as soon as later today, but definitely Monday.

**Comment by iamjoshfrank** [comment]
>> Community. In a remote-first world, people want to be seen, be recognized for their good work. we need to leverage that desire for attention and affirmation. posting the content must be its own reward. fucked up as it is, social media nurtures a dependence for a single dose of attention to keep folks coming back.

This problem/opportunity is definitely one I'm poking around at, too. I think there's really something "there" there, though I wonder if it's more about pumping teammates up and shouting them out than it is about recording snapshots of my own day's accomplishments / my next day's plans.

I might be too hung up on "what information is typically being communicated within a typical Standup; and why that information is important."

Maybe Hot Takes aren't really serving those same informational ends or striving to create the same kind of value. Unsettled.

I also wonder about the video mechanics of the Hot Take, but I'm trying not to let my doubts about "willingness to adopt" interfere with how it can help blow open a much wider range of possibilities. (I can't quit picturing the droves of engineers that prefer command line tools over GUIs, for instance). But that's dumping on an idea that's still new and raw and I don't want to do that prematurely or ungenerously, 😂 .

---

What is the nature of the social / emotional / community connection that teammates want to form while working, especially remotely? How is it similar to and different from the kinds of connections they're striving to make in non-work settings? 🤔

**Comment by ackernaut** [comment]
It truly does help to see how you folks are thinking through this. I’ve been hung up on establishing guiding principles for our product for a while now. This idea and conversation prompted [some writing, I’ll share it here](https://www.notion.so/parabol/Team-activities-emergent-value-000c0f7cec1646b5a0d6915010cb53f6) in the spirit of transparent thinking.

**Comment by iamjoshfrank** [comment]
Random thought connected to:

> how would you describe the setting for this idea (the setting / scenario you imagine users being in)?

>> unspecified. we need to avoid coming off like range.co or a slackbot. nagging users with "what are you working on?" feels wrong. the billion dollar question is how do we encourage users to generate content without their managers telling them to.

and

>> we need to leverage that desire for attention and affirmation. posting the content must be its own reward. fucked up as it is, social media nurtures a dependence for a single dose of attention to keep folks coming back.

I'm thinking a bit longitudinally here, and maybe prematurely. I'm trying to consider, "what might motivate someone to create a Hot Take?" or "how might a Daily Standup not have to be triggered by a reminder?"

I'm thinking about what situations might prompt a Hot Take - what the person is doing just before, during, and after.

And I'm thinking about what might encourage habit building.

I'm also curious about aversion to reminders. I've seen them work and not felt nagged. Maybe I need to dig into that personal experience to understand why they felt not-horrible and actually-useful to mine for the ore.

__(I'm thinking about a lot; probably too much. But at least the messy beginning has begun!)__

**Comment by avivapinchas** [comment]
> I'm also curious about aversion to reminders. I've seen them work and not felt nagged. Maybe I need to dig into that personal experience to understand why they felt not-horrible and actually-useful to mine for the ore.

Just one person's perspective, but what feels annoying to me about a daily standup reminder is: 
- Daily (or weekly) is a kind of arbitrary length of time - the size of tasks may or may not fit into that time, but there's a pressure to come up with a new thing because we're at the next reminder. 
- The reminder doesn't serve _me_ - I have other practices for remembering tasks. This reminder is usually so I aggregate my tasks in a way to serve a manager, or package it together for a team. If I actually wanted to remember to share, I'd create my own reminder, in the way that worked for me. 

🤔 I wonder if a contextual prompt to share an HT™️ would feel better? If I move a task to 'done' that had been lingering for a long time, or maybe is particularly long, maybe then I get the suggestion to share. As I build that habit and share more, I get fewer prompts to do this, because I'm already doing it organically.

From a manager's perspective, the time I've found folks eager to share what they were working on is when they knew an all-hands meeting was coming up and I'd be sharing what our team was doing with the broader org. They wanted the opportunity for their work to be visible on that stage. 🤔 So maybe we surface that a manager is making a highlight reel, and ask the team member if they want to submit something?

**Comment by mattkrick** [comment]
> So maybe we surface that a manager is making a highlight reel, and ask the team member if they want to submit something?

this feels right. just like a manager opens a meeting so people can reflect, a manager can open a highlight reel & you can add your update. we can proto this with a check-in meeting. you can add your updates async. 

> If I move a task to 'done' that had been lingering for a long time, or maybe is particularly long, maybe then I get the suggestion to share.

i like this, but today folks don't log in enough/use the kanban to have this be the right place for a trigger. as DAU increases, this will be the right answer.

**Comment by ackernaut** [comment]
>this feels right. just like a manager opens a meeting so people can reflect, a manager can open a highlight reel & you can add your update. we can proto this with a check-in meeting. you can add your updates async.

👾 👾 👾

**Comment by shawneeatparabol** [comment]
Really loving this from so many angles. A highlight reel to take tasks off of managers plates is fantastic, adding multimedia capabilities addresses the common desire I hear from customers to add more inspirational/creative aspects to their meetings. The drive for affirmation is high and this will draw the users that like affirmation to our platform more. This is also super unique and adds to our current value of 'creating compassion and nurturing best practices' in such an engaging way. 

A couple thoughts/questions:
-Automation is key. Prompts to add completed tasks to a Hot Take, easy integrations, quick editing/automated editing, etc. If this takes more than 2 minutes a day I think many people will see it as just another additional task rather than a cool perk.

-Yes, Stand-Ups are about packaging your tasks to serve a team/manager. But the biggest value add of stand-ups for me as a team member is highlighting where you're stuck and being able to address issues in a relaxed cadence rather than ad hoc (so that workloads aren't disrupted). I would want to keep this aspect in the meeting structure. I'm unclear how Hot Takes would affect that part of the meeting, if at all.

-What about people who aren't doing well, are super shy, don't like attention? How can we create this in a way that is inviting to those who would appreciate it but not intimidating to those that would prefer not to partake? How do we keep Parabol a safe space once accomplishments start getting upvoted and shared externally?

**Comment by gcrickman** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/21482951) 🔒  expressed an interest in standup meetings as a meeting type on Parabol, they currently use Slack for their standups.

**Comment by ackernaut** [comment]
This is superseded by newer issues for standups like #4771
